### DISCUSSION TITLE
Add/Releases as Community Packages to Package Center

### DISCUSSION THREAD

**Discussion by Frank-D-HB** [bb-qq/r8152#420] [Ideas]
Reason/Arguments/Motivation:
     belonging to my expericence, many users just want click and have fun, and not start a shh session, retry installation, as described at
     the installation procedure
Benefits:
     easy install/use of the package
     may be less installation issues/support needed
     more visibility to more "non power" users
btw: there other USB drivers eg. USB serial and USB CD-Rom drivers offered as "Community Packages" at Package Center, so i think the usb network drivers would be a good addtion.

**Comment by L0rdLogan** [comment]
I do not know exactly what the command does, but I guess it's adding the package identifier to some sort of trusted list, which I don't think would be possible on the community packages, it's similar to installing an unsigned driver on windows, it doesn't like it, so you have to tell it it's okay

**Comment by Frank-D-HB** [comment]
Hallo,

thanks for your fast response.

I am just a amateur, but i see (and use) other usb-packages there (SynoCommunity <https://synocommunity.com/packages> ), like USB serial- kerner driver package, or USB -CD-Rom kernel driver package and i gues that these are also „not signed“ by synology or the cd-rom manufactures.

May be you can ask the synology-community „package-provider“ (SynoCommunity · GitHub <https://github.com/SynoCommunity> ), what are the prequesites/conditions/technic to publish your package there? Because i am not able to ask them the right questions.

I am impressed by these usb-packages, how easy they are to install (just click), automatic updates inclusive, and also to uninstall them again.

Kind regards,

Frank 

 

Von: L0rdLogan ***@***.***> 
Gesendet: Sonntag, 11. August 2024 12:13
An: bb-qq/r8152 ***@***.***>
Cc: Frank-D-HB ***@***.***>; Author ***@***.***>
Betreff: Re: [bb-qq/r8152] Add/Releases as Community Packages to Package Center (Discussion #420)

 

I do not know exactly what the command does, but I guess it's adding the package identifier to some sort of trusted list, which I don't think would be possible on the community packages, it's similar to installing an unsigned driver on windows, it doesn't like it, so you have to tell it it's okay

**Comment by bb-qq** [comment]
Thanks for the suggestion.

I am not familiar with what is in the USB serial driver, but this r8152 driver emphasizes safe installation without rewriting the system as much as possible. Changing distribution method while maintaining the current mechanism would probably require a lot of work, so I have no plans to do that now.
